### PR TITLE
Only set sasl_user when output_mode is rhev

### PR DESCRIPTION
**Pull request by xiaodwan**
Signed-off-by: xiaodwan <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.convert_from_file.positive_test.linux.input_mode.ova.normal.ova_tar.output_mode.libvirt: PASS (212.61 s)

**Comment by xiaodwan** [comment]
@chloerh please help to review it.

**Comment by xiaodwan** [comment]
@chunfuwen please help to review it, Thank you!

File: v2v/tests/src/convert_from_file.py [hunk 1]
```diff
@@ -45,9 +45,10 @@ def run(test, params, env):
     error_list = []
 
     # create different sasl_user name for different job
-    params.update({'sasl_user': params.get("sasl_user") +
-                   utils_misc.generate_random_string(3)})
-    logging.info('sals user name is %s' % params.get("sasl_user"))
+    if output_mode == 'rhev':
```
Comment (chunfuwen) [inline v2v/tests/src/convert_from_file.py]: Does it mean that if output_mode is not equal to rhev, sasl_user will not be set?

Comment (xiaodwan) [reply v2v/tests/src/convert_from_file.py]: yes, sasl_user will only be set in ovirt hosts. so the output_mode must be 'rhev'.

**Pull request merged**